src/benchcab/fluxsite.py

```diff
@@ -60,6 +61,20 @@ def __init__(
         self.sci_config = sci_config
         self.logger = get_logger()
 
+    def _get_state_dir(self) -> Path:
+        return internal.STATE_DIR / "fluxsite" / "runs" / self.get_task_name()
+
+    def _state_init(self):
+        state_dir = self._get_state_dir()
+        state_dir.mkdir(parents=True, exist_ok=True)
+
+    def _state_set(self, attr: str):
+        (self._get_state_dir() / attr).touch()
+
+    def is_done(self) -> bool:
+        """Return status of current task."""
+        return (self._get_state_dir() / "done").exists()
```

Comment:
Change to is_file() - exists will return True for dir or file. Not likely to happen, but could be a pain in future when cleaning up the state directory.

Reply:
Marking this as resolved for now as the implementation has changed.